src/backend/routers/student.ts

```diff
@@ -86,5 +86,25 @@ export const student = router({
       return result;
     }),
 
+  // TODO: get only the active IEP from the student
```

Comment:
it looks like this condition currently holds?

Reply:
I believe this condition will hold for our MVP since we only need to retrieve one student iep.